### PR TITLE
Describe nullability of columns in the create table schema endpoint

### PR DESCRIPTION
**Pull request by dsalv**

### CODE REVIEW

**Comment by dsalv** [comment]
* **#106** <a href="https://app.graphite.dev/github/pr/duneanalytics/dune-api-docs/106?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @dsalv and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: api-reference/tables/endpoint/create.mdx [hunk 1]
```diff
@@ -24,7 +33,7 @@ curl --request POST \
   "table_name":"interest_rates",
   "description": "10 year daily interest rates, sourced from https://fred.stlouisfed.org/series/DGS10",
   "is_private": false,
-  "schema": [{"name": "date", "type": "timestamp"}, {"name": "dgs10", "type": "double"}]
+  "schema": [{"name": "date", "type": "timestamp", "nullable": false}, {"name": "dgs10", "type": "double"}]
```
Comment (waddahAldrobi) [inline api-reference/tables/endpoint/create.mdx:36]: Can you align this schema, with rest of the schema definition's 👇

**Review by hawkaa — CHANGES_REQUESTED** [review]
We've forgotten to add this to the actual openapi schema. When that is done, we shouldn't need to write out a separate text paragraph describing the same thing.

**Comment by 0xBoxer** [comment]
@duneanalytics/team-produce are you guys on this or are you watining on dev rel?

**Comment by dsalv** [comment]
> @duneanalytics/team-produce are you guys on this or are you watining on dev rel?

Both works

**Review by a-monteiro — APPROVED** [review]

File: api-reference/tables/tables-openapi.json [hunk 2]
```diff
@@ -373,6 +373,11 @@
             ],
             "description": "The column type.",
             "example": "timestamp"
+          },
+          "nullable": {
+            "type": "boolean",
+            "description": "Flag that denotes if the column is nullable (true/false, true by default).",
+            "example": "false"
```
Comment (helanto) [inline api-reference/tables/tables-openapi.json:380]: Shouldn't this be:
```suggestion
            "example": false
```

**Review by helanto — COMMENTED** [review]
A comment about the openAPI schema

File: api-reference/tables/endpoint/create.mdx [hunk 3]
```diff
@@ -12,6 +12,15 @@ The resulting table will be empty, and can be inserted into with the [/insert en
 - To delete a table, you can go to `user settings (dune.com) -> data -> delete` or use the [/delete endpoint](./delete).
 </Note>
 
+## Schema
```
Comment (waddahAldrobi) [inline api-reference/tables/endpoint/create.mdx:15]: Why this paragraph? 

I would just modify this
image

**Review by waddahAldrobi — CHANGES_REQUESTED** [review]

File: api-reference/tables/endpoint/create.mdx (hunk 3 above)
Comment (0xBoxer) [reply api-reference/tables/endpoint/create.mdx:15]: I'd think this helps users tbh, it's a nice and quick intro

**Comment by 0xBoxer** [comment]
this lgtm from my perspective now, we can debate whether the text should be there or not, I think it helps to grog this on a quick glance.

**Pull request merged**